settingshelper.py

```diff
@@ -282,3 +284,10 @@ def get_git_commit(base_dir):
         return out.strip().decode('ascii')
     except OSError:
         pass
+
+
+# Reference: https://django-simple-captcha.readthedocs.io/en/latest/advanced.html#roll-your-own
+def random_alphanumeric_challenge():
```

Comment:
that is great, its nice how straight forward this was.
Any reason you have taken uppercase and digits as opposed to just digits?

Reply:
Requirement by AKS was 6 digit Alphanumeric captcha. So I had to take both